pyproject.toml

```diff
@@ -5,7 +5,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "checkpoint_schedules"
-version = "1.0.0"
+version = "1.0.2"
```

Comment:
Not sure on conventions for versions in the pyproject.toml -- does this need to be changed yet?

Reply:
It is necessary because Pypi does not accept update wheels without changing the package version.